parameters:
    camera_num: 2
    calibration_file_path: "param.yaml" # only .yaml file
    mono_source_dirs: ["first", "second"]
    stereo_source_dirs: ["first", "second"]
    iou_threshold: 0.5
    min_accepted_calib_error: 0.7
    required_board_coverage: 0.7
    marker_color: [0.0, 1.0, 0.0, 0.12] #  [R, G, B, A], each in range 0.0-1.0
